src/main/webapp/app/entities/project/project-detail.component.html

```diff
@@ -1,59 +1,113 @@
+<div id="wrapper" [ngClass]="{'toggled': true}">
+    <div id="sidebar-wrapper">
+        <div class="sidebar-nav" *ngIf="project">
+            <p class="status-header">
+                {{project.projectName}}
+            </p>
+            <ul class="status-list">
+                <ng-container>
+                    <li [ngClass]="{'status-group-selected': showSubjects , 'status-group': !showSubjects}"
```

Comment:
I think we should add a `cursor: pointer` CSS to these list items so it is more obvious to the user they are clickable

Reply:
Good point. Done!